src/github.rs

```diff
@@ -957,20 +957,28 @@ impl RequestSend for RequestBuilder {
     }
 }
 
+fn validate_token(t: &str) -> bool {
+    t.chars().all(|char| char.is_digit(16))
```

Comment:
Do we have docs for this?

Reply:
mmm ... I'm afraid I'm missing what you mean :-)

Do you mean we should link the documentation on [how to create this token](https://docs.github.com/en/free-pro-team@latest/actions/reference/authentication-in-a-workflow) or that this assumption about the token format can fail at any time (and thus this patch should just be closed)?

Reply:
I don't really see a reason we need to validate the token to make use of it -- ultimately, the only real validation is GitHub :)

But I would prefer not to validate it ourselves if we don't have evidence the tokens are always hexadecimal.

Reply:
Fair enough. The main argument for this patch in invalid, so I'll close it as such.

Thanks for the review